src/namespaces/PlaybackNamespace.js

```diff
@@ -189,9 +195,8 @@ export default class PlaybackNamespace extends GMusicNamespace {
             // Make sure that this is the first of the notifications for the
             // insertion of the song information elements.
             if (!currentTrack.equals(lastTrack)) {
-              this.emit('change:track', currentTrack);
```

Comment:
Just to make it consistent with the other ones. I can revert it if you want though.